third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -127,6 +127,48 @@ impl ScriptCache {
     }
 }
 
+pub(crate) struct StructNameCache {
```

Comment:
Could we move this to some utility file and add a few really basic unit tests for interfaces?